_includes/index.md

```diff
@@ -6,7 +6,7 @@ A campaign for Mozillians around the world to have impact in areas key to Mozill
 ## Focus Initiatives
 
 {: #area-deveng .area}
-[![image](/asserts/img/development.png)
+[![image]({{ "/asserts/img/development.png" | prepend: site.baseurl }})
```

Comment:
Locally at my computer, the image shows up without this change (both in the English version as well as on http://127.0.0.1:4000/es/ ). Could you please explain what exactly you run into here?